shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { StyledToolTipProps } from '../../../lib/types';
+import { fonts } from '../../../style/typography';
+
+const StyledToolTipText = styled.div`
+display: inline-block;
+text-align: left;
+line-height: 2em;
+font-family: ${fonts.nunitoSansSemibold};
+`;
+
+const StyledToolTip = styled.div<StyledToolTipProps>`
+  width: 12em;
+  background-color: ${(props): string => (props.team === 'Directors' ? '#F2C94C': 
```

Comment:
Can you make an object or something thats like the colors object?
This will make it less prone to string / spelling errors if you check props.team === 'teams.directors'
const teams = {
directors: "Directors",
tech: "Tech",
...
}